src/CMakeLists.txt

```diff
@@ -268,13 +268,8 @@ list(APPEND SM_WINDOWS_PROGRAM_DLLS
   "${SM_PROGRAM_DIR}/avcodec-53.dll"
   "${SM_PROGRAM_DIR}/avformat-53.dll"
   "${SM_PROGRAM_DIR}/avutil-51.dll"
-  "${SM_PROGRAM_DIR}/msvcp100.dll"
```

Comment:
Recent releases on Windows are built with VS2013. These files are VS2010/2012 runtime libraries, doing absolutely no help, thus should be removed. Now that /MT flag is enabled, I'd like to expect users to install corresponding redistributable packages (if further releases are still built /MTd) to solve DLL problems, instead of putting in newer DLLs.